File: 4_SERIES/Series_09.java
/*
 * Juez Patito ID: 2169 Serie: 1 3 6 11 18 19 22 29 38 41 50 51 ...
 */
import java.util.Scanner;

public class Series_09 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("N: ");
        int N = sc.nextInt();
        int p = -2;
        int a = 1;
        System.out.print(a + " ");
        
        for (int j = 0; j < N; j++) {
            int cont = 0;
            for (int i = 1; i < N; i++) {
                if (p % i == 0) {
                    cont = cont + 1;
                }
            }
            if (cont == 2) {
                System.out.print(a + p % 10 + " ");
                a = a + p % 10;
                j = j + 1;
                p = p + 1;
            } else {
                p = p + 1;
            }
        }
    }
}

File: 4_SERIES/Series_09.py
'''
Juez Patito ID: 2169 Serie: 1 3 6 11 18 19 22 29 38 41 50 51 ...
'''
N=int(input("N: "))
p=-2; a=1
print(a,end=" ")
for j in range(N):
    j=1; cont=0
    for i in range(1,N):
        if p%i==0:
            cont=cont+1
    if cont==2:
        print(a+p%10,end=" ")
        a=a+p%10
        j=j+1
        p=p+1
    else:
        p=p+1